doc/TROUBLESHOOTING.md

```diff
@@ -97,3 +97,32 @@ System's local adb and Nix's adb differ. As adb include of server/client process
 Always use respective `make` commands, e.g. `make android-ports`, `make android-devices`, etc.
 
 Alternatively, run adb commands only from `make shell TARGET=android` shell. Don't forget the `TARGET=android` env var setting - otherwise `adb` will still be selected from the system's default location. You can double-check this by running `which adb`.
+
+# Hot Reloading Crashes
+## APP Crashes on reloading changes
+
+### Cause
+Status-mobile uses watchman for monitoring changes and uses its own [reloader](https://github.com/status-im/status-mobile/blob/develop/src/status_im/reloader.cljs) for updating the running app. If react-native's fast refresh is also enabled then it creates conflicts and crashes the app.
```

Comment:
Status-mobile doesn't use watchman, fast refresh uses, but it should be disabled

Reply:
or do you know that shadow-cljs uses watchman ?

Reply:
but anyway better to say that status-mobile uses shadow-cljs for hotreload

Reply:
Hi @flexsurfer, Thank you for reviewing PR.

> or do you know that shadow-cljs uses watchman ?

Not sure, but according to this [stackoverflow question](https://stackoverflow.com/questions/71771072/react-native-fast-refresh-works-without-watchman), fast refresh also worked without watchman.

> but anyway better to say that status-mobile uses shadow-cljs for hotreload

yeah, this sounds better. updated, thank you